feat: add `constants/lapack/dsafmin`

Pranavchiku opened the pull request:
Towards #2464.

## Description

> What is the purpose of this pull request?

This pull request adds macro for constant [`dsafmin`](https://www.netlib.org/lapack/explore-html/d4/d69/namespacela__constants_a7ae97fc519ab76a576db113b17312382.html#a7ae97fc519ab76a576db113b17312382).

## Related Issues

> Does this pull request have any related issues?

NA

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

Pranavchiku committed:
feat: add constants/lapack/dsafmin

Pranavchiku committed:
fix: license year in readme file

Planeshifter commented on lib/node_modules/@stdlib/constants/lapack/dsafmin/README.md:
```diff
@@ -0,0 +1,153 @@
+
+#### FLOAT64_LAPACK_SAFE_MIN
+
+The minimum [safe][safe] [double-precision floating-point][ieee754] integer.
```
I believe the description should be changed here and elsewhere to e.g.

```suggestion
The smallest positive [safe][safe] [double-precision floating-point][ieee754] number.
```

aman-095 replied:
Applied changes.

aman-095 commented:
/stdlib merge

stdlib-bot committed:
Merge remote-tracking branch 'upstream/develop' into dsafmin

aman-095 committed:
docs: apply review changes

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: passed
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: passed
  - task: lint_javascript_tests
    status: passed
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: passed
  - task: lint_typescript_tests
    status: passed
  - task: lint_license_headers
    status: passed
---

---
type: pre_push_report
description: Results of running various checks prior to pushing changes.
report:
  - task: run_javascript_examples
    status: passed
  - task: run_c_examples
    status: na
  - task: run_cpp_examples
    status: na
  - task: run_javascript_readme_examples
    status: passed
  - task: run_c_benchmarks
    status: na
  - task: run_cpp_benchmarks
    status: na
  - task: run_fortran_benchmarks
    status: na
  - task: run_javascript_benchmarks
    status: na
  - task: run_julia_benchmarks
    status: na
  - task: run_python_benchmarks
    status: na
  - task: run_r_benchmarks
    status: na
  - task: run_javascript_tests
    status: passed
---
